demos/sev-demo/README.md

```diff
@@ -0,0 +1,136 @@
+    ├── modules.order
+    ├── modules.softdep
+    ├── modules.symbols
+    └── modules.symbols.bin
+```
+
+Make sure that the module matches your kernel version. Speaking of which, `efi_secret` also requires some minor kernel patches. These patches and the module are not currently upstream. You will need to find Dov Murik's EFI secret patches on the kernel mailing list and build a kernel with them. This should also give you the correct version of the `efi_secret` module.  
+
+
+## Kata Runtime
+
+Since SEV and SEV-ES use pre-attestation driven from outside of the guest, some changes are required for the Kata runtime. These changes are not currently upstream. You will need to build your Kata runtime with [this PR](https://github.com/kata-containers/kata-containers/pull/3025). This will work best with QEMU 6.2.0rc0. Newer versions of QEMU may require you to set `kernel-hashes=on` via the Kata runtime.
+
+You will also need to set some parameters in the Kata configuration file usually located at `/etc/kata-containers/configuration.toml`.
+
+First, you'll need to enable SEV via the `confidential_guest` parameter.
+```
+# Enable confidential guest support.
+# Toggling that setting may trigger different hardware features, ranging
+# from memory encryption to both memory and CPU-state encryption and integrity.
+# The Kata Containers runtime dynamically detects the available feature set and
+# aims at enabling the largest possible one.
+# Default false
+confidential_guest = true
+```
+
+Next you'll need to enable pre-attestation.
+```
+# Enable guest attestation
+# (default: false)
+guest_attestation=true
+#
+# Guest owner proxy that handles remote attestation
+guest_attestation_proxy="localhost:50051"
+#
+# Preattestation keyset ID for injected secrets
+guest_attestation_keyset="KEYSET-1"
+```
+Here, we are specifying a local KBS. You can also run the KBS remotely. For this demo we specify the `keyset` in the configuration file. In the future this will be overridden via a Kubernetes annotation. The keyset parameter will be described more below. For now, you can just leave it set to `KEYSET-1`. 
+
+
+## KBS 
+
+You can use the `offline_sev_kbc` manually by injecting a specially formatted json file, but you will probably want to use our [key broker server](https://github.com/confidential-containers-demo/scripts/tree/main/guest-owner-proxy) instead. Note that the SEV KBS is often referred to as a Guest Owner Proxy (GOP). This was our original terminology. First, install `gop-client.py` on the worker node by placing the file at `/opt/sev/guest-owner-proxy/gop-client.py`. You should check the `gop-client.py` file and make sure that the `hw_api_major`, `hw_api_minor`, and `hw_build_id` match the parameters reported by the AMD-SP. You can verify these parameters using [SEV-Tool](https://github.com/AMDESE/sev-tool). These parameters will be set automatically in the future and the functionality of `gop-client.py` may be added to the Kata runtime.
```

Comment:
- s/json/JSON/
- I like the options you're giving readers but would it be simpler to just say "do this, then do this, and then this happens?" That would make maintenance of this document simpler as there is only one "code path" through the doc.